vector.cabal

```diff
@@ -40,9 +40,9 @@ Tested-With:
   GHC == 8.2.2,
   GHC == 8.4.4,
   GHC == 8.6.5,
-  GHC == 8.8.1,
-  GHC == 8.10.1
-
+  GHC == 8.8.4,
+  GHC == 8.10.3
```

Comment:
Why not 8.10.4?

Reply:
Because CI matrix uses 8.10.3. But good point I should update this